eth/tracers/api.go

```diff
@@ -333,7 +324,7 @@ func (api *API) traceChain(start, end *types.Block, config *TraceConfig, closed
 			close(resCh)
 		}()
 		// Feed all the blocks both into the tracer, as well as fast process concurrently
-		for number = start.NumberU64(); number < end.NumberU64(); number++ {
+		for number = start.NumberU64() + 1; number <= end.NumberU64(); number++ {
```

Comment:
This test case has been changed: number is now the block that we are scheduling a tracing task for.  The reason for this change is that I separate block prestate retrieval into a separate method that is reused in other areas of this file, and this block number takes the block that we want to retrieve a prestate for (parent block is no longer explicitly needed in this function).